modules/tensor_mechanics/examples/coal_mining/cosserat_elastic.i

```diff
@@ -0,0 +1,569 @@
+# Strata deformation and fracturing around a coal mine
+#
+# A 2D geometry is used that simulates a transverse section of
+# the coal mine.  The model is actually 3D, but the "x"
+# dimension is only 10m long, meshed with 1 element, and
```

Comment:
Oh yea, the LayeredCosserat that i use is also only for 3D - i forgot about that.